doc/api/animation_api.rst

```diff
@@ -32,7 +32,8 @@ to.  If you do not hold a reference to the `Animation` object, it (and
 hence the timers) will be garbage collected which will stop the
 animation.
 
-To save an animation to disk use `Animation.save` or `Animation.to_html5_video`
+To save an animation to disk use `Animation.save`, `Animation.to_html5_video`,
```

Comment:
```suggestion
To save an animation use `Animation.save`, `Animation.to_html5_video`,
```

I see @timhoffm 's point, but eventually the html will end up on disk _someplace_.  I think it is saver to drop 'disk' .